packages/web-components/src/components/va-text-input/va-text-input.tsx

```diff
@@ -312,12 +313,12 @@ export class VaTextInput {
               {messageAriaDescribedby}
             </span>
           )}
-          {!charcount && maxlength && value?.length >= maxlength && (
-              <span class={messageClass} aria-live="polite">
+          {isMessageDisplayed && !charcount && maxlength && value?.length >= maxlength && (
```

Comment:
Originally I was thinking we could just do this with css `display: none` but that doesn't remove the element from the DOM. Adding this to the display condition removed it completely.